src/entities/settings/SettingsChangeResponse.java

```diff
@@ -0,0 +1,19 @@
+package entities.settings;
+
+public class SettingsChangeResponse {
+  private final String error;
+  private final Boolean isError;
+
+  public SettingsChangeResponse(String error, Boolean isError) {
```

Comment:
This should only take a String error, and `isError` should return `error != null`. At least get rid of this `getIsError` stuff